wazimap_ng/points/views.py

```diff
@@ -171,3 +171,12 @@ def list(self, request, profile_id, theme_id=None):
 
         return Response(data)
 
+
+class ThemeList(generics.ListAPIView):
+    queryset = models.Theme.objects.all()
+    serializer_class = serializers.ThemeSerializer
+
+    def get_queryset(self):
+        qs = super().get_queryset()
+        profile_id = self.request.query_params.get('profile_id', None)
```

Comment:
@goyal1092 why are you accepting the profile_id on the querystring, shouldn't it be in the URL to be consistent with the rest of the API? Also, I think you should throw a 404 the profile is missing

Reply:
@adieyal We already have a url for : `api/v1/points/profile/<int:profile_id>/themes/` that points to theme_view
I just want to avoid any confusion between api endpoints.
I will update PR to throw 404 when there is invalid/nonexistent profile_id.

Reply:
@adieyal I have updated the PR.

Reply:
I am not sure I am following. What this the difference between

api/v1/points/profile/<int:profile_id>/themes/

and

api/v1/points/themes/

Reply:
Hello @adieyal 
* Theme_view  is called for url : api/v1/points/profile/int:profile_id/themes/
It uses profile category to get themes for api.
So we intentionally exclude any theme that is not linked to profile category yet.
https://github.com/OpenUpSA/wazimap-ng/blob/master/wazimap_ng/points/views.py#L48
I didn't want to touch any existing api code. so I made a new api endpoint.

* ThemeList is called for url : api/v1/points/themes/
It just returns all themes or return themes for a specific profile_id.